FAQ.md

```diff
@@ -0,0 +1,82 @@
+# FAQ
+
+## How do I run the test suite?
+
+`mvn test` will run unit tests while `mvn verify` will include integration tests and benchmarks.
+
+See [the testing documentation](/TESTING.md) for additional notes about running the test suite.
+
+## How do Accounts/Usernames/Subusers differ?
+
+Joyent's SmartDataCenter account implementation is such that you can have a
+subuser as a dependency upon a user. This is part of SmartDataCenter's [RBAC
+implementation](https://docs.joyent.com/public-cloud/rbac/users). A subuser
```

Comment:
You don't define users or accounts, you've jumped straight to subusers. Consider breaking it down:

> Joyent's SmartDataCenter account implementation is such that you can have a subuser as a dependency upon a user. This is part of SmartDataCenter's [RBAC implementation](https://docs.joyent.com/public-cloud/rbac/users).
> 
> + **Accounts** - The group created for your company on Triton, managed by a user and subusers.
> + **Users** - Users are login credentials that allow different people in your organization to log in to your Joyent Cloud account. Generally, users are account owners, but this term may be used for account owners and in place of "subusers".
> + **Subusers** - A subuser belongs to the account holder, who maintains control of the subusers. This may be people in your organizations who belong to the larger company account. Subusers have a unique username that is joined with the account holder's username. This is in the format of "user/subuser".

I think the definition for "accounts" definitely could warrant more thought -- I may come back to this. Especially considering "accounts" as thought of for Joyent are not necessarily the same as seen in the Java Manta library.